geoips/plugins/modules/readers/seviri_hrit.py

```diff
@@ -275,7 +274,7 @@ def get_latitude_longitude(gmd, BADVALS, area_def):
     ne.evaluate("Rs - (sn * cos_x * cos_y)", out=s1)
 
     # Nothing unneed, no inplace
-    s2 = ne.evaluate("sn * sin_x * cos_y")
+    s2 = ne.evaluate("sn * sin_x * cos_y")  # noqa: F841
```

Comment:
Make a note as to why we needed these ignores.  Also - it may not be required where the variables are USED within the string, but only where the seemingly unused variable is SET?

Reply:
added comment in this function explaining those tags